app/javascript/mastodon/components/filter_warning.tsx

```diff
@@ -0,0 +1,37 @@
+import { FormattedMessage } from 'react-intl';
+
+export const FilterWarning: React.FC<{
```

Comment:
This is duplicating most of `<ContentWarning>`, maybe the two of them could be specialised versions of a base component?